packages/components/src/internal/components/productnavigation/utils.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { TEST_USER_APP_ADMIN, TEST_USER_EDITOR } from '../../userFixtures';
+import { TEST_LIMS_STARTER_MODULE_CONTEXT, TEST_LKSM_STARTER_MODULE_CONTEXT } from '../../productFixtures';
+
+import { shouldShowProductNavigation } from './utils';
+
+describe('shouldShowProductNavigation', () => {
+    test('LKB non-premium admin user', () => {
+        let moduleContext = { ...TEST_LIMS_STARTER_MODULE_CONTEXT, api: { applicationMenuDisplayMode: 'ADMIN' } };
+        expect(shouldShowProductNavigation(TEST_USER_APP_ADMIN, moduleContext)).toBeTruthy();
```

Comment:
Is there value in this checking `toBeTruthy` as opposed to `toBe(true)`?

Reply:
No, not to my knowledge. For boolean values, I assume those checks are pretty much equivalent. I'm happy to change this if you feel strongly.